dsgrid/config/dataset_config.py

```diff
@@ -252,76 +258,106 @@ class DatasetConfigModel(DSGBaseDatabaseModel):
+    origin_organization: str | None = Field(
+        default=None,
         title="origin_organization",
-        description="Origin organization name, e.g., NREL",
+        description="Organization name of the origin_creator, e.g., 'NREL'",
     )
-    origin_contributors: list[str] = Field(
+    origin_contributors: list[str] | None = Field(
+        default=None,
         title="origin_contributors",
-        description="List of origin data contributor's first and last names"
-        """ e.g., ["Harry Potter", "Ronald Weasley"]""",
-        default=[],
+        description=(
+            "List of contributors to the compilation of this dataset for dsgrid, "
+            " e.g., ['Harry Potter', 'Ronald Weasley']"
+        ),
     )
-    origin_project: str = Field(
+    origin_project: str | None = Field(
+        default=None,
         title="origin_project",
-        description="Origin project name",
-    )
-    origin_date: str = Field(
-        title="origin_date",
-        description="Date the source data was generated",
+        description=(
+            "Name of the project for/from which this dataset was compiled, e.g., "
+            "'IEF', 'Building Standard Scenarios'."
+        ),
     )
-    origin_version: str = Field(
-        title="origin_version",
-        description="Version of the origin data",
+    user_defined_metadata: dict[str, Any] = Field(
+        title="user_defined_metadata",
+        description="Additional user defined metadata fields",
+        default={},
     )
-    source: str = Field(
-        title="source",
-        description="Source of the data (text description or link)",
+    tags: list[str] | None = Field(
+        default=None,
+        title="tags",
+        description="List of data tags",
     )
+    # ETH@20251008 - Although we could define a default DataClassificationType,
+    # it seems better to default to 'low' by priniting in the template, so that
+    # the base assumption of low risk is clear to dataset contributors.
     data_classification: DataClassificationType = Field(
         title="data_classification",
-        description="Data security classification (e.g., low, moderate, high)",
+        description="Data security classification (e.g., low, moderate).",
         json_schema_extra={
             "options": DataClassificationType.format_for_docs(),
         },
     )
-    tags: list[str] = Field(
-        title="source",
-        description="List of data tags",
-        default=[],
-    )
     enable_unit_conversion: bool = Field(
         default=True,
         description="If the dataset uses its dimension mapping for the metric dimension to also "
         "perform unit conversion, then this value should be false.",
     )
     # This field must be listed before dimensions.
     use_project_geography_time_zone: bool = Field(
-        default=False,
+        default=False,  # ETH@20251008 - Default here conflicts with make_unvalidated_dataset_config
```

Comment:
Do we need to change something?

Reply:
They don't actually conflict anymore. Removing the comment.